cloud-formation/membership-app.cf.yaml

```diff
@@ -59,6 +59,18 @@ Mappings:
       DynamoDBTables:
```

Comment:
I know it's not been introduced by this PR, but I don't think membership-frontend actually needs any information about Promotions these days...